accessscopes.go

```diff
@@ -0,0 +1,28 @@
+package goshopify
+
+type AccessScopesService interface {
+	List(interface{}) ([]AccessScope, error)
+}
+
+type AccessScope struct {
+	Handle string `json:"handle,omitempty"`
+}
+
+// AccessScopesResource represents the result from the oauth/access_scopes.json endpoint
+type AccessScopesResource struct {
+	AccessScopes []AccessScope `json:"access_scopes,omitempty"`
+}
+
+// AccessScopesServiceOp handles communication with the Access Scopes
+// related methods of the Shopify API.
```

Comment:
Suggesting for consistency - the other comments in this file don't end with a period.
```suggestion
// related methods of the Shopify API
```

Reply:
Fixed that up, thanks for the comment. Take another look when you can!